projects/fsastorefrontstyles/scss/overrides/main-overrides.scss

```diff
@@ -64,3 +64,14 @@ cx-fs-cms-custom-container {
   padding: 0;
   box-shadow: none;
 }
+
+header .SiteLinks {
+  @include media-breakpoint-down(md) {
+    background-color: inherit;
+    position: absolute;
```

Comment:
Another `position: absolute` (fourth)? Please try to refactor this approach.

Reply:
Also please check if this impacts greeting message in hamburger menu on mobile as well?

Reply:
No, because hamburger menu is in the PreHeader slot